### PR TITLE
bump for the sdk / fixed .js.map file for sdk.min

### PR DESCRIPTION
**Pull request by ocke**
@Mehul-Porwal noted that `grunt bump` used to be an available command, so we put that back.

Also added source mapping for `zaf_sdk.min.js` file

`npm run bump`

@zendesk/vegemite

### CODE REVIEW

**Comment by fvzzy** [comment]
Also squash your commits with a message that adheres to these guidelines, please https://chris.beams.io/posts/git-commit/#seven-rules.

**Comment by keitheous** [comment]
👀

File: README.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ Then run:
 
 `npm install` - Install dependencies
 
-`npm run build` - Build
+`npm run build` - Build production
+
+`npm run build:dev` - Build development version of the sdk. This doesn't minify nor transform
```
[the remark this answers is not in the document]
Comment (ocke) [reply README.md]: You're right, it does (babel) transform.

**Comment by ocke** [comment]
@fvzzy addressed your comments.

**Pull request merged**